ddl/cluster.go

```diff
@@ -176,6 +180,9 @@ func checkAndSetFlashbackClusterInfo(sess sessionctx.Context, d *ddlCtx, t *meta
 	if err = setTiDBSuperReadOnly(d.ctx, sess, variable.On); err != nil {
 		return err
 	}
+	if err = setTiDBTTLJobEnable(d.ctx, sess, variable.Off); err != nil {
```

Comment:
As we have turned off the `TTL_ENABLE` for every table, is it still necessary to turn off this variable?

Reply:
for flashback cluster，we did not turn off any table's TTL_ENABLE and just set `tidb_ttl_job_enable='OFF'`